pkgs/tools/backup/rdiff-backup/default.nix

```diff
@@ -1,23 +1,36 @@
-{lib, fetchurl, python2Packages, librsync, gnused }:
+{ lib, python3Packages, librsync }:
 
-python2Packages.buildPythonApplication {
-  name = "rdiff-backup-1.3.3";
+let
+  pypkgs = python3Packages;
```

Comment:
If you add ``with python3.pkgs; `` to the callPackages function in all-packages.nix we can just use buildPythonApplication, fetchPypi, etc. as inputs.